README.md

```diff
@@ -5,10 +5,64 @@
 
 ## Installation
 
+
 ```
 yarn add --dev jest-styled-components
 ```
 
+This package must be used with either `react-test-renderer` or `enzyme` and `enzyme-to-json` to generate a serialized `tree` from your component.
+
+**Using react-test-renderer**
+
+
+Installation:
+
+```
+yarn add --dev react-test-renderer
+```
+
+Usage:
+
+```js
+import renderer from 'react-test-renderer'
+
+// somewhere in your tests
+it('matches the styled components snapshot', () => {
+  const tree = renderer.create(<MyComponent />).toJSON()
+  expect(tree).toMatchStyledComponentsSnapshot()
+  expect(tree).toHaveStyleRule('display', 'block')
+})
+```
+
+For more info on using react-test-renderer with jest for snapshots see [this](https://facebook.github.io/jest/docs/snapshot-testing.html)
+
+
+**Using enzyme's mount and enzyme-to-json**
+
+Installation:
+
+```
+yarn add --dev enzyme enzyme-to-json
+```
+
+Usage:
+
+This module is not compatible with shallow rendering, components must use the full DOM rendering with `mount`
+
+```js
+import {mount} from 'enzyme'
+import toJSON from 'enzyme-to-json'
```

Comment:
`enzyme-to-json` can be used like this in each test file, or it can be [added globally](https://github.com/styled-components/jest-styled-components/blob/master/package.json#L54) (which is more convenient).